minor updates

CeoFred opened the pull request:
- Change DB name to something better
- my_modules had a function without a block, fixed that
- waiting for a model to show up to continue sending in more PR

CeoFred committed:
minor updates

EphraimDev commented:
There are no reall changes in this commit. I'll merge when you make problem solving commits

CeoFred committed:
conflict resolved

CeoFred committed:
add company route

CeoFred commented:
Updated with things that matter
- Added route to get all companies
- Added helper functions to help return consistent api response strucuture
- Fix router issues that broke the app
- Remove unused code that tried to initiate a user model that does not exist
- A few refactoring to keep ote
